docker/influxdb/influxdb.conf

```diff
@@ -0,0 +1,153 @@
+reporting-disabled = false
```

Comment:
This looks like a complete conf file. Is there not a way to extend a conf instead of having a whole file?

I'm concerned this will go stale over time.

Reply:
I can override the values with environment variables. I'll remove the conf and do that instead.